renovate-default.json

```diff
@@ -152,41 +153,43 @@
         "github.com/longhorn/longhorn-engine",
         "github.com/longhorn/longhorn-instance-manager",
         "github.com/longhorn/longhorn-share-manager",
-        "github.com/longhorn/backing-image-manager"
+        "github.com/longhorn/backing-image-manager",
+        "github.com/longhorn/cli"
       ],
-      "allowedVersions": "/^v1\\.7\\.\\S+/",
+      "allowedVersions": "/^v1\\.8\\.\\S+/",
       "matchManagers": [
         "gomod"
       ],
       "groupName": "longhorn branch repo dependencies"
     },
     {
       "matchBaseBranches": [
-        "v1.6.x"
+        "v1.7.x"
       ],
       "matchPackageNames": [
         "github.com/longhorn/longhorn-engine",
         "github.com/longhorn/longhorn-instance-manager",
         "github.com/longhorn/longhorn-share-manager",
-        "github.com/longhorn/backing-image-manager"
+        "github.com/longhorn/backing-image-manager",
+        "github.com/longhorn/cli"
       ],
-      "allowedVersions": "/^v1\\.6\\.\\S+/",
+      "allowedVersions": "/^v1\\.7\\.\\S+/",
       "matchManagers": [
         "gomod"
       ],
       "groupName": "longhorn branch repo dependencies"
     },
     {
       "matchBaseBranches": [
-        "v1.5.x"
+        "v1.6.x"
```

Comment:
The EOL of 1.5 is 17 Nov 2024, so we still keep it until it reaches EOL.

Reply:
Updated. Thank you.